docs/user-guide/rug/command-handlers.md

```diff
@@ -0,0 +1,239 @@
+side the project they target or in different projects altogether if they are generic. 
+In both cases however, Rug command handlers share the same approach as other Rugs. 
+For the  purpose of this documentation, we will assume our handler implementations
+live in their own projects rather than in an existing project as the documentation
+describes for Rug [generators][] or [editors][].
+
+{!handler-layout.md!}
+
+Rug commands are declared in Rug command handlers. Command handlers handle
+commands coming from the Atomist bot. The next section describes a basic Rug
+command handler implementation.
+
+## Example Command Handler
+
+Suppose we want to open a new GitHub issue:
+
+```typescript linenums="1"
+import {
+    HandleCommand, 
+    MappedParameters, 
+    HandlerContext , 
+    CommandPlan} from '@atomist/rug/operations/Handlers'
+import {
+    CommandHandler, 
+    Secrets, 
+    MappedParameter, 
+    Parameter, 
+    Tags,
+    Intent} from '@atomist/rug/operations/Decorators'
+import {Issue} from '@atomist/cortex/Issue'
+
+@CommandHandler("CreateGitHubIssue", "Create an issue on GitHub")
+@Tags("github", "issues")
+@Secrets("github://user_token?scopes=repo")
+@Intent("create issue")
+class CreateIssueCommand implements HandleCommand {
+
+    @Parameter({description: "The issue title", pattern: "^.*$"})
+    title: string
+
+    @Parameter({description: "The issue body", pattern: "^.*(?m)$"})
+    body: string
+
+    @MappedParameter(MappedParameters.GITHUB_REPOSITORY)
+    repo: string
+
+    @MappedParameter(MappedParameters.GITHUB_REPO_OWNER)
+    owner: string
+
+    @MappedParameter("atomist://correlation_id")
+    corrid: string
+
+    handle(ctx: HandlerContext): CommandPlan {
+        let plan = new CommandPlan();
+        plan.add(
+            { 
+                instruction: { 
+                    kind: "execute", 
+                    name: "create-github-issue",
+                    parameters: this
+                },
+                onError: { 
+                    kind: "respond", 
+                    name: "GenericErrorHandler", 
+                    parameters: this 
+                }
+            }
+        )
+        plan.add(handleErrors(exec, this))
+        return plan;
+    }
+}
+
+export let create = new CreateIssueCommand()
+```
+
+[events]: event-handlers.md
+
+This command handler follows the same programming model as other Rugs, so it 
+should look familiar.
+
+### Declaration
+
+The first lines group the Rug typing imports which, as usual, provide interfaces
+and decorators to implement and declare your handlers. The 
+`#!typescript UpdateChangeLog` class expresses how the command is invoked, as well
+as how to handle success and error scenarios. `#!typescript ChangeLogEditorSuccess`
+and `#!typescript HandleUpdateChangeLogError` are Response Handlers to handle the
+success and failure responses of the `#!typescript UpdateChangeLog` command handler
+respectively.
```

Comment:
I'd drop the word respectively here as I think the subject of the sentence has changed. You would need to use it after success and failure if you want to keep it (I think)